css/style.css

```diff
@@ -6,3 +6,55 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+  body {
+background-color: rgb(227, 237, 241);
+margin: 2em;
+padding: 1rem;
+display: flex;
+flex-direction: column;
+ }
+
+ h1{
+color: black;
```

Comment:
Is it necessary to set the heading colour to black? What would happen if you removed this rule?